machineconfiguration/v1/custom.machineconfigpool.testsuite.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "[Custom] MachineConfigPool"
+crd: 0000_80_machineconfigpool-TechPreviewNoUpgrade.crd.yaml
+tests:
+  onCreate:
+  - name: Should be able to create a minimal MachineConfigPool
+    initial: |
+      apiVersion: machineconfiguration.openshift.io/v1
+      kind: MachineConfigPool
+      spec:
+        pinnedImageSets:
+          - name: test-pinnedimageset
```

Comment:
This isn't a required field is it? Generally the `minimal` version of this test shows only required fields